src/components/form/Navigation.tsx

```diff
@@ -36,12 +36,15 @@ export default function Navigation({
   const { gotoPrevQuestion, gotoNextQuestion, noPrevQuestion, noNextQuestion } =
     useForm()
 
-  const { isMissing, plancher, value } = useRule(question)
+  const { isMissing, plancher, plafond, value } = useRule(question)
 
   const { updateCurrentSimulation } = useCurrentSimulation()
 
   const isNextDisabled =
-    tempValue !== undefined && plancher !== undefined && tempValue < plancher
+    (tempValue !== undefined &&
```

Comment:
Petite sudj, ici je trouve ça pas méga lisible et j'ai l'impression qu'on peut éviter une répétition, j'ajouterai bien des coms, dis moi ce que tu en penses !

```suggestion
    tempValue !== undefined &&
    // value is below the floor
    ((plancher !== undefined && tempValue < plancher) ||
    // value is above the ceiling
    (plafond !== undefined && tempValue > plafond))
```

Reply:
plutôt tenir compte de mon autre com en priorité